applications/GeoMechanicsApplication/custom_constitutive/incremental_linear_elastic_law.h

```diff
@@ -21,27 +21,27 @@ namespace Kratos
 class ConstitutiveLawDimension;
 
 /**
- * @class GeoLinearElasticPlaneStrain2DLaw
+ * @class GeoIncrementalLinearElasticLaw
  * @ingroup GeoMechanicsApplication
- * @brief This class defines a small deformation linear elastic constitutive model for plane strain cases
+ * @brief This class defines a incremental linear elastic constitutive model for plane strain and 3D cases
```

Comment:
Very minor (nitpicking at this point)
```suggestion
 * @brief This class defines an incremental linear elastic constitutive model for plane strain and 3D cases
```